frontend/src/metabase/static-viz/components/XYChart/utils/margin.ts

```diff
@@ -2,7 +2,7 @@ import { measureTextHeight } from "metabase/static-viz/lib/text";
 import { CHART_PADDING } from "metabase/static-viz/components/XYChart/constants";
 import { ChartSettings } from "metabase/static-viz/components/XYChart/types";
 
-export const LABEL_OFFSET = 10;
+export const LABEL_OFFSET = 12;
```

Comment:
Make this the same number as `CHART_PADDING` in `frontend/src/metabase/static-viz/components/XYChart/constants.ts`